gradle/libs.versions.toml

```diff
@@ -75,6 +76,7 @@ lombok = { group = "org.projectlombok", name = "lombok", version.ref = "lombok"
 junit-jupiter-api = { group = "org.junit.jupiter", name = "junit-jupiter-api", version.ref = "junit" }
 junit-jupiter-params = { group = "org.junit.jupiter", name = "junit-jupiter-params", version.ref = "junit" }
 junit-jupiter-engine = { group = "org.junit.jupiter", name = "junit-jupiter-engine"}
+junit4 = { group = "junit", name = "junit", version.ref = "junit4"}
```

Comment:
The initialization of `HDFSMiniCluster` of `Hadoop 2` depends on `Junit4`, otherwise it will cause its initialization to fail, so it needs to be introduced in the test.